addons/link_tracker/models/link_tracker.py

```diff
@@ -316,7 +316,7 @@ class LinkTrackerClick(models.Model):
 
     campaign_id = fields.Many2one(
         'utm.campaign', 'UTM Campaign', index='btree_not_null',
-        related="link_id.campaign_id", store=True, ondelete="set null")
+        default="link_id.campaign_id", store=True, ondelete="set null")
```

Comment:
Not sure why this change, but this seems louche ^^ . Not sure it even works as intended.

Reply:
@tde-banana-odoo good, point, i wanted to re-check this for a long time. i have changed code lot of times and maybe this is leftover code (will take a look soon)

Reply:
@tde-banana-odoo it seems this one was leftover code, therefore I removed branch. (I tested code thoroughly and also logically, didn't find any issue)